applications/nrf5340_audio/src/bluetooth/bt_content_control/content/bt_media_control.h

```diff
@@ -14,8 +14,7 @@
  *
  * @param  play Differentiate between play command and pause command.
  */
-typedef void (*ble_mcs_play_pause_cb)(bool play);
-
+typedef void (*bt_media_control_play_pause_cb)(bool play);
 
 /**
  * @brief  Discover MCS and included services.
```

Comment:
```suggestion
 * @brief  Discover MCS and included services, only valid for client.
```

And remove ", only valid for client." from @param